.github/workflows/test_build.yml

```diff
@@ -0,0 +1,59 @@
+name: Test Build
+
+on:
+  push:
```

Comment:
We can update it to run on pull request, too, right? Or is that unnecessary when you're using `push`?

Reply:
It seems that way. This implementation is pretty liberal. It will run anytime a push is made. We can restrict it to only run on pushes to `main` and then have it run on all pull_requests. The pull_request event will be fired each time someone pushes to an open PR.

Reply:
I'm going to go ahead and update it to use this strategy instead, but let me know if you have more thoughts @mattstauffer

Reply:
Nope looks good to me!